### ISSUE TITLE
Crash on Ctrl-D (EOF)

### ISSUE THREAD

**Issue by MediaServe** [neo-project/neo-cli#47]
When typing `CTRL-D` (stty EOF) in the terminal, the program crashes with core dump.

```
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Neo.Services.ConsoleServiceBase.RunConsole()
   at Neo.Services.ConsoleServiceBase.Run(String[] args)
   at Neo.Program.Main(String[] args)
Aborted (core dumped)
```

**Expected behaviour:**
Exit the command line interface

**Comment by erikzhang** [comment]
This is a bug of .NET Core 1.x, and has been fixed in .NET Core 2.0.
BTW, to exit the command line interface, just type `exit`.

**Comment by ThomasLobker** [comment]
According to the changelog this issue has been fixed in `neo-cli` version `2.9.0`.
